src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -322,4 +397,37 @@ private function createOrder(
 
         return $order;
     }
+
+    /**
+     * @param $count
+     *
+     * @return CustomerInterface[]
+     */
+    private function generateCustomers($count)
+    {
+        $customers = [];
+
+        for ($i = 0; $i < $count; $i++) {
+            $customer = $this->customerFactory->createNew();
+            $customer->setEmail(sprintf('john%s@doe.com', uniqid()));
```

Comment:
`uniqid()` -> `$i`? Just to be consistent with the lastname (and platform / time independent)